ee/kernel/src/timer_alarm.c

```diff
@@ -89,7 +89,7 @@ static int timOverflow(int ca) {
 
     // Check if we actually triggered an alarm
     if (event_queue) {
-        if (ps2_clock() > (event_queue->scheduled_time >> 8)) {
+        if (GetTimerSystemTime() > (event_queue->scheduled_time)) {
```

Comment:
Have all references to the legacy ps2_clock been replaced by the new GetTimerSystemTime?

Reply:
Now, yes (at least in `ps2sdk`)